corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap3/solutions-feature-request.js

```diff
@@ -0,0 +1,107 @@
+hqDefine('hqwebapp/js/bootstrap3/solutions-feature-request', [
```

Comment:
So this is the one same as for reporting an issue.

Reply:
This one was similar in that it was mostly copied from the JS file for reporting an issue. I have since changed this though so that both modals now use the same generic JS file using knockout.